lib/msal-common/src/client/RefreshTokenClient.ts

```diff
@@ -36,20 +37,24 @@ export class RefreshTokenClient extends BaseClient {
         responseHandler.validateTokenResponse(response.body);
         return await responseHandler.handleServerTokenResponse(
             response.body,
-            this.authority
+            this.authority,
+            "",
+            "",
```

Comment:
These should perhaps be null instead of empty strings.

Reply:
Semantically, both are the same (falsey). If we pass null, strictnullChecks would require the type to be `string | null`, which  results in downstream work to account for that.

Reply:
Makes sense. I would argue that empty strings and null are two separate things, and that these values here are actually null as opposed to empty string and so the precision is worth the extra work. Ultimately its a nitpick either way, since you mentioned we are using StringUtils.isEmpty(), so functionally it doesn't matter.

Reply:
I'll add that null is already used in some other places, so whichever one is picked should be consistent throughtout.

Reply:
Fixed in the latest push, no longer sending empty/null params